src/behaviors/Visibility/Visibility.js

```diff
@@ -141,7 +142,7 @@ export default class Visibility extends Component {
   }
 
   static defaultProps = {
-    context: window,
+    context: isBrowser ? window : {},
```

Comment:
if we use null, should we also add null checks around https://github.com/ChrisNLott/Semantic-UI-React/blob/d3bd18cd7fca2be2ea5a32381b70798aa5de2c57/src/behaviors/Visibility/Visibility.js#L173?